src/lib/builtins/mod.rs

```diff
@@ -1,3 +1,16 @@
+/// Macro to create a new member function of a prototype
+/// If no length is provided, the length will be set to 0.
+macro_rules! make_builtin_fn {
+    ($fn:ident, named $name:expr, with length $l:tt, of $p:ident) => {
```

Comment:
Is there a reason why `$l` is a `tt`?

Reply:
Because it’s the best way to represent numbers in macros